lib/utils/device/deploy.ts

```diff
@@ -121,6 +126,58 @@ async function environmentFromInput(
 	return ret;
 }
 
+async function buildArgumentsFromInput(
```

Comment:
Basically a copy paste of `environmentFromInput()`, done under the assumption that we should support setting build arguments for specific services. If this is the way to go, we should probably break out the shared logic.